package.json

```diff
@@ -21,5 +21,11 @@
   "bugs": {
     "url": "https://github.com/tigercosmos/node-color-log/issues"
   },
-  "homepage": "https://github.com/tigercosmos/node-color-log"
+  "homepage": "https://github.com/tigercosmos/node-color-log",
+  "devDependencies": {
+    "@types/node": "^18.11.8"
```

Comment:
it should be able to support even older versions, since we don't rely on fancy new features.

Reply:
Which version of nodejs should we target?

Reply:
maybe v12?
I meant, there is no reason to restrict the nodejs version.
we didn't use those new features in new NodeJS.

Reply:
ok, we don't restrict it in any way, it's nothing more than hints for the code editor.
I think version 12 would be the most appropriate.